src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,46 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({
+  movie: { title, description, imgUrl, imdbUrl },
+}) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={imgUrl}
+          alt="Film logo"
```

Comment:
Alt attribute for image tag is not descriptive. It should describe the content of the image. Replace 'Film logo' with something like {movie.title} logo.